kedro-datasets/tests/spark/test_spark_dataset.py

```diff
@@ -440,6 +442,12 @@ def test_copy(self):
         assert spark_dataset_copy._file_format == "csv"
         assert spark_dataset_copy._save_args == {"mode": "overwrite"}
 
+    @pytest.mark.parametrize("filepath", ["data", "/data", "dbfs/data", "/dbfs/data"])
+    def test_dbfs_filepath_prefix(self, filepath, mocker):
+        mocker.patch.dict(os.environ, {"DATABRICKS_RUNTIME_VERSION": "7.3"})
```

Comment:
```suggestion
    def test_dbfs_filepath_prefix(self, filepath, monkeypatch):
        monkeypatch.setenv( "DATABRICKS_RUNTIME_VERSION", "7.3")
```
Just for consistency with other tests (see e.g. `test_rich_traceback_disabled_on_databricks` and several others).

Reply:
Thanks, made this change.